src/scenes/DeliverablesRouter/QuestionsDeliverableView.tsx

```diff
@@ -224,7 +227,7 @@ const QuestionsDeliverableView = (props: Props): JSX.Element => {
       <Page crumbs={crumbs} rightComponent={actionMenu} title={<TitleBar {...props} />}>
         {(props.isBusy || requestStatus === 'pending') && <BusySpinner />}
         <Box display='flex' flexDirection='column' flexGrow={1}>
-          <QuestionsDeliverableStatusMessage {...viewProps} questions={ppsSearchResults?.data || []} />
+          {/* <QuestionsDeliverableStatusMessage {...viewProps} questions={ppsSearchResults?.data || []} /> */}
```

Comment:
Just commented out for brevity and since the `ppsSearchResults` are no longer here.